src/screens/ProfileScreen/elements/FollowInfoRow.js

```diff
@@ -60,12 +60,12 @@ let styles = StyleSheet.create({
   following: {marginLeft: 18},
   textTotal: {
     fontSize: 14,
-    color: colors.bondi_blue,
+    color: COLORS.blue,
     paddingRight: 4
   },
   textFollow: {
     fontSize: 14,
-    color: colors.black,
+    color: COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values for `textTotal` and `textFollow` styles have been updated. The color for `textTotal` has been changed from `colors.bondi_blue` to `COLORS.blue`, and the color for `textFollow` has been changed from `colors.black` to `COLORS.black`. Make sure these new color constants are defined in the `COLORS` object and they match the intended colors for these elements.
```diff
-    color: colors.bondi_blue,
+    color: COLORS.blue,

-    color: colors.black,
+    color: COLORS.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->